src/model/fsm_util.jl

```diff
@@ -195,7 +180,7 @@ function populate_fsm!{NumType <: Number}(
                             mv_calculate_derivs,
                             mv_calculate_hessian,
                             gal_mcs[k, j, i, s], x, wcs_jacobian,
-                            is_active_source)
+                            active_source)
```

Comment:
Let's use `is_active_source` here to indicate that it's a boolean value, rather than an element of `active_sources` (an array of ints, not booleans).